metrics/prometheus.go

```diff
@@ -135,6 +135,14 @@ func NewPrometheusCollector(i infoProvider, f ContainerLabelsFunc) *PrometheusCo
 					return metricValues{{value: float64(time.Now().Unix())}}
 				},
 			}, {
+				name:      "container_restart_count",
+				help:      "Counter of times a container has been restarted",
+				valueType: prometheus.CounterValue,
+				getValues: func(s *info.ContainerStats) metricValues {
+					//return metricValues{{value: float64(11111)}}
```

Comment:
The idea here is to make available the times a countainer has been restarted to Prometheus. This is why I've created this metric (and also this PR), so we can check and alert here when a container have a big restart count.

Is this really necessary to be removed?